src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -18,72 +20,159 @@ import { FIGMA_URL } from '../../utils/common';
   }}
 />
 
-# Tooltip
+<SectionHeader title={'Tooltip'} />
```

Comment:
i would expect also hover for chromatic screenshots 😄 like we did in select I think